Sources/Gui/Main.cpp

```diff
@@ -57,13 +57,23 @@
 #endif
 
 
+#ifdef _MSC_VER>=1900	// Visual Studio 2015 or higher
+extern "C" { FILE __iob_func[3] = { *stdin,*stdout,*stderr }; }
```

Comment:
I checked the filesystem (the directories it was loading stuff from, and of them was the directory where the .exe was, was, so I copied the Resources folder from the release (the .pak's) *and* the source to that dir, and it worked! The game run perfectly fine. For some reason it didn't look for the folder `OpenSpades.msvc/os.Resources` which the REAME instructs to put the .pak's in. Did I use Cmake wrong? I just did `cmake .`
Thanks for the help man, awesome project you got there.